server/services/diskUsageService.js

```diff
@@ -8,7 +8,8 @@ const execFile = util.promisify(require('child_process').execFile);
 const config = require('../../config');
 const diskUsageServiceEvents = require('../constants/diskUsageServiceEvents');
 
-const PLATFORMS_SUPPORTED = ['darwin', 'linux'];
+const PLATFORMS_SUPPORTED = ['darwin', 'linux', 'freebsd'];
```

Comment:
```diff
-freebsd
+bsd
```

As it is Linux and not a distro name.

Reply:
Sure but process.platform returns the string ‘freebsd’. Also I wouldn’t assume the output of df is consistent across all BSDs. technically Darwin is a BSD and the output is different :P

Reply:
@inve1 You are totally right, my bad.